plugins/user-settings/src/components/AuthProviders/ProviderSettingsItem.tsx

```diff
@@ -72,7 +77,9 @@ export const ProviderSettingsItem = ({
         primary={title}
         secondary={
           <Tooltip placement="top" arrow title={description}>
-            <span>{description}</span>
+            <span>
+              {description}. Signed in as {profileEmail}
```

Comment:
I'd prefer if the profile info was on a row of its own, perhaps standing out a little using colour or boldness or something. Could also have more than just the email (do some smart string building based on what the provided parts of the profile actually were, perhaps in a helper function).

Maybe this could be something like

```tsx
<Grid container spacing={3} alignItems="stretch">
  <Grid item xs={12} md={6}>
    {description}
  </Grid>

  <Grid item xs={12} md={6}>
    {profileInfo)
  </Grid>
</Grid>
```

or something like that. Completely untested, just an idea :) Then the profile info would be between the description and the button, and could even contain a div with several lines of stuff in it if you wanted

Reply:
Would say that we want to display all the information as well if it's available. Display name, email, and avatar. I think it would be fine to leave that for a later PR too though